packages/gatsby/src/bootstrap/index.js

```diff
@@ -72,16 +73,7 @@ module.exports = async (args: BootstrapArgs) => {
   // Try opening the site's gatsby-config.js file.
   activity = report.activityTimer(`open and validate gatsby-config.js`)
   activity.start()
-  let config
-  try {
-    // $FlowFixMe
-    config = preferDefault(require(`${program.directory}/gatsby-config`))
-  } catch (err) {
-    if (!testRequireError(`${program.directory}/gatsby-config`, err)) {
```

Comment:
Unless I'm a crazy person, I think this was a bug actually. `testRequireError` returns `true` in the example when gatsby-config.js can't be found or is misnamed.

Reply:
No it's correct. `testRequireError` only returns an error if `gatsby-config.js` exists *and* has a JS error of some sort. Not having a config file is fine.